crates/wasm/lib.rs

```diff
@@ -323,7 +305,22 @@ impl WasmRuntime {
     &mut self,
     state: &mut [u8],
     action: &mut [u8],
+    interaction_context: Value,
   ) -> Result<Vec<u8>, AnyError> {
+    let contract = deno_core::serde_json::to_vec(&interaction_context)?;
+
+    let (mut contract, contract_ptr_high, contract_len) = {
+      let scope = &mut self.rt.handle_scope();
+      let undefined = v8::undefined(scope);
+      let alloc_obj = self.allocator.open(scope).to_object(scope).unwrap();
+      let alloc = v8::Local::<v8::Function>::try_from(alloc_obj)?;
```

Comment:
we're already opening the allocator below (in the `let result` block). This logic can be there